docker/README.md

```diff
@@ -1,9 +1,8 @@
 ## SETUP
-1. `echo "MESOS_SOURCES=/home/user/mesos-upstream/" > .env`
-2. `docker build -f docker/Dockerfile --target runner -t runner-mesos .`
-3. `docker build -f docker/Dockerfile --target builder -t builder-mesos .`
-4. Then, from MESOS_SOURCES dir: `docker run -v $(pwd):/src/mesos builder-mesos`
-5. Go back in mesosbuild and `docker-compose up`
+1. `docker build -f docker/Dockerfile --target runner -t runner-mesos .`
+2. `docker build -f docker/Dockerfile --target builder -t builder-mesos .`
+3. Then, from root dir: `docker run -v $(pwd):/src/mesos builder-mesos`
+4. Go back in `docker/` folder and `docker-compose up`
```

Comment:
Maybe we could advise using Docker BuildKit which speeds up quite a bit the process.
Also why not use `docker-compose -f docker/docker-compose.yml up` ?

Reply:
You are right

Reply:
But I don't know what is BuildKit, can you elaborate a bit please?

Reply:
Docker BuildKit is the new and improved builder from docker.
It allows for clever copy from build context and overall speeds up a lot the build.
It can be activated like this for instance: `DOCKER_BUILDKIT=1 docker build .`
https://docs.docker.com/develop/develop-images/build_enhancements/

Reply:
So, is this not something that should be up to the user? 
I'll add it in the README anyway, as an example.

Reply:
Well using or not the new buildkit is up to the user yes, i fell like it's a good place to advertise it since it saves quite a bit of time